client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-site-metrics.tsx

```diff
@@ -1,39 +1,42 @@
 import { useIsEnglishLocale } from '@automattic/i18n-utils';
 import { getQueryArg } from '@wordpress/url';
 import { useUrlBasicMetricsQuery } from 'calypso/data/site-profiler/use-url-basic-metrics-query';
-import { upgradePlanSiteMetricsLcpThreshold } from '../constants';
+import {
+	upgradePlanSiteMetricsLcpThreshold,
+	upgradePlanSiteMetricsFidThreshold,
+} from '../constants';
+
+const calcPercentageDifferenceFromThreshold = ( value: number | undefined, threshold: number ) => {
+	if ( ! value || value <= threshold ) {
+		return 0;
+	}
+
+	return Math.round( 100 * Math.abs( ( value - threshold ) / ( ( value + threshold ) / 2 ) ) );
+};
 
 export const useGetUpgradePlanSiteMetrics = () => {
 	const isEnglishLocale = useIsEnglishLocale();
 	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
 	const { data: siteMetricData } = useUrlBasicMetricsQuery( importSiteQueryParam );
-	let showUpdatedSpeedMetrics = false;
+
 	let lcpPercentageDifference = 0;
+	let fidPercentageDifference = 0;
 
-	if (
-		isEnglishLocale &&
-		siteMetricData?.basic?.lcp &&
-		siteMetricData?.basic?.lcp > upgradePlanSiteMetricsLcpThreshold
-	) {
-		lcpPercentageDifference =
-			( siteMetricData?.basic?.lcp &&
-				Math.round(
-					100 *
-						Math.abs(
-							( siteMetricData?.basic?.lcp - upgradePlanSiteMetricsLcpThreshold ) /
-								( ( siteMetricData?.basic?.lcp + upgradePlanSiteMetricsLcpThreshold ) / 2 )
-						)
-				) ) ||
-			0;
+	if ( isEnglishLocale ) {
+		lcpPercentageDifference = calcPercentageDifferenceFromThreshold(
+			siteMetricData?.basic?.lcp,
+			upgradePlanSiteMetricsLcpThreshold
+		);
 
-		if ( lcpPercentageDifference > 0 ) {
-			showUpdatedSpeedMetrics = true;
-		}
+		fidPercentageDifference = calcPercentageDifferenceFromThreshold(
+			siteMetricData?.basic?.fid,
+			upgradePlanSiteMetricsFidThreshold
+		);
 	}
 
 	return {
-		showUpdatedSpeedMetrics,
```

Comment:
Removed this in order to simplify this hook, otherwise, we'd need 2 values per type of metric.